lua/fidget/integration/nvim-tree.lua

```diff
@@ -39,7 +39,7 @@ require("fidget.options").declare(M, "integration.nvim-tree", M.options, functio
       local col = vim.api.nvim_win_get_position(winid)[2]
       if col > 1 then
         local width = vim.api.nvim_win_get_width(winid)
-        win.set_x_offset(width)
+        win.set_x_offset(width + 1)
```

Comment:
What's this for? Any constant offsets should be handled by the `notification.window.x_padding` setting, since some folks use scrollbars, and/or just prefer more space. (But was the offset broken, i.e., off-by-one, before?)

Reply:
Yes, the offset is off-by-one, but probably fixing the math will require fixing multiple places? So I can either revert it or leave it for now and you can try fixing math and the removing this +1.

Reply:
It's because the `nvim_win_get_width` gets the width of the text buffer area, but does not include the border column. I think it makes sense to leave in this adjustment, since the notion of having a border is specific to nvim-tree and xcodebuild popping open windows that have borders.